### DISCUSSION TITLE
Run multiple jobs at a same time.

### DISCUSSION THREAD

**Discussion by veed76** [laravel/framework#32948] [General]
- Laravel Version: 7.0
- PHP Version: 7.2.5
- Database Driver & Version:MySQL & 5.7.26

### Description:
Suppose I have 10 jobs in the jobs table and I have want to run these 10 jobs at a same time, so using the Laravel queue job is it possible to run multiple jobs at the same time in the background? and it's a good way to run multiple jobs at a same time?

`QUEUE_CONNECTION=database`

The 10 jobs are just, for example, there is a chance there will be 100,150, etc, job to run at a same time.

Otherwise, can anyone suggest me the good way?
Ex: https://prnt.sc/smwpkm

Thank You.

**Comment by netpok** [comment]
You can start multiple queue workers, but for 100 instance you need a considerable amount of ram.

Also jobs are not running in the background but on different process(es) based on your workload you can start a few workers and it will handle them one after other. As long as your jobs are finished in your deadline and your queue isn't bottlenecked you should be good.

**Comment by driesvints** [comment]
Moving this to discussions.

**Comment by xcesaralejandro** [comment]
Some update about this?

**Reply by netpok** [reply]
Well, each worker handles one job at the time, if you want to handle more jobs simultaneously just start more workers. Also the last time I used database queue driver it was not very stable with more than few workers so personally I would use the redis driver.

**Reply by xcesaralejandro** [reply]
@netpok  Thanks for your recommendation, I might think that the database driver could be more stable. Can you tell me a little more about what you call stability? it would be of great use to me.

**Reply by netpok** [reply]
There were deadlock issues but the last time I used it was like in 5.6 so maybe it is fixed then.
